### DISCUSSION TITLE
How do I a Profile Image and icons Proxy Hosts?

### DISCUSSION THREAD

**Discussion by Invictus6474** [NginxProxyManager/nginx-proxy-manager#3830] [Q&A] [answered]
I can't, for the life of me, figure out how to change the profile picture or add icons to the Proxy Hosts.
Please help!

![Nginx Proxy Hosts Icons](https://github.com/NginxProxyManager/nginx-proxy-manager/assets/99221192/4a69c433-da0f-4165-af24-9d09617a786a)

**Comment by muelleraschheim** [comment, marked as answer]
The page is using Gravatar (https://www.gravatar.com) for the configured e-mail of the logged-in user.
Sign up for free with Gravatar, connect your e-mail with your picture and you're done...

**Reply by Invictus6474** [reply]
Ohhh.. Ok.. I would've loved to have icons relevant to each service on these.
Thank you!
